### PR TITLE
Implement RESTful service

### PR DESCRIPTION
**Pull request by yura182**

### CODE REVIEW

**Comment by yura182** [comment]
Swagger pull request https://github.com/yura182/rest-service/pull/2

File: src/main/java/com/yura/resthw/contorller/OrderController.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("order")
+public class OrderController {
```
Comment (valeriiafatova) [inline src/main/java/com/yura/resthw/contorller/OrderController.java:19]: Use plural

Comment (valeriiafatova) [reply src/main/java/com/yura/resthw/contorller/OrderController.java:19]: where method get all?

File: src/main/java/com/yura/resthw/contorller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("users")
+public class UserController {
```
Comment (valeriiafatova) [inline src/main/java/com/yura/resthw/contorller/UserController.java:19]: How possible to get all orders of user?

**Comment by yura182** [comment]
> Swagger pull request #2

@valeriiafatova

File: src/main/java/com/yura/resthw/entity/UserEntity.java [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "users")
+public class UserEntity {
```
Comment (valeriiafatova) [inline src/main/java/com/yura/resthw/entity/UserEntity.java:14]: users/1/orders/5

orders/5
users/5

**Pull request merged**